homeassistant/components/homewizard/button.py

```diff
@@ -0,0 +1,57 @@
+"""Support for HomeWizard buttons."""
+
+import logging
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import HWEnergyDeviceUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Identify button."""
+    coordinator: HWEnergyDeviceUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    features = await coordinator.api.features()
+    if features.has_identify:
+        async_add_entities([HomeWizardIdentifyButton(coordinator, entry)])
+
+
+class HomeWizardIdentifyButton(
+    CoordinatorEntity[HWEnergyDeviceUpdateCoordinator], ButtonEntity
+):
+    """Representation of a identify button."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: HWEnergyDeviceUpdateCoordinator,
+        entry: ConfigEntry,
+    ) -> None:
+        """Initialize button."""
+        super().__init__(coordinator)
+        self._attr_unique_id = f"{entry.unique_id}_identify"
+        self._attr_device_info = {
```

Comment:
Not every entity needs to have device info. You could just link it up to the main one by just providing the `identifiers` key.

Reply:
Awesome, did not know that. Fixed in https://github.com/home-assistant/core/pull/82375/commits/914109f6de10b9f93685bed7e24bdceaeeeaa476